wordle.py

```diff
@@ -74,11 +107,46 @@ def play(word):
 
         if guess.lower() == word:
             print('WOW YOU WON')
-            return
-    print("SORRY YOU LOST. The WORD was", word)
+            return 1
+    print("SORRY YOU LOST.")
 
 
 if __name__ == '__main__':
-    random_word = get_random_word("words.txt")
-    print(random_word)
-    play(random_word)
+    print("Starting...")
+    CHOSEN_WORD = get_random_word("words.txt")
+
+    if len(sys.argv) > 1:
+        if len(sys.argv[1]) == 5:
+            CHOSEN_WORD = sys.argv[1]
+        if sys.argv[1] == "--hint-mode":
+            HINT_MODE = True
+
+    synonyms, antonyms, definitions, examples = ([] for i in range(4))
```

Comment:
can you do same with `[] * 4`?

Reply:
Nope. This simply returns an empty list. Even If we do [[]]*4 we get [[], [], [], []] but all the 4 lists have the same underlying reference. Thus any changes in one of them result in changes to all of them.